spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -80,32 +104,72 @@ definitions:
             type: CarrierPhase
             units: cycles
             desc: Carrier phase observation with typical sign convention.
+        - D:
+            type: Doppler
+            units: Hz
+            desc: Doppler observation with typical sign convention.
         - cn0:
             type: u8
-            units: dB Hz * 4
+            units: dB Hz * 0.25
```

Comment:
does it make more sense to use `dB Hz / 4` thats what tracker status message is using